app/assets/stylesheets/administrate/base/_tables.scss

```diff
@@ -16,6 +16,10 @@ td {
 
 td {
   border-bottom: $base-border;
+  font-feature-settings: "kern", "liga", "clig", "calt", "lnum", "tnum";
+  font-kerning: normal;
+  font-variant-ligatures: common-ligatures, contextual;
+  font-variant-numeric: lining-nums, tabular-nums;
```

Comment:
It’s real. I think Firefox uses it: https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-numeric

I think I might be able to tell SCSS-Lint that this is an acceptable property within it’s config…